servers:
  get_semantic_map:
    service: "~get_map"
